GW2EIEvtcParser/ParserHelpers/EffectGUIDs.cs

```diff
@@ -7,6 +7,8 @@ namespace GW2EIEvtcParser
 {
     public static class EffectGUIDs
     {
+        // Generic
+        public const string Generic600UnitRadius = "9C7C1B2379CCDD4990001A38030E4495";
```

Comment:
What is that?

Reply:
Effect displaying a radius indicator of 600 units. It's used for several skills and pulsing effects. Thought it might be useful to note down for future reference, so you can recognize and ignore it when looking into effects for a skill.